app/styles/_variables.scss

```diff
@@ -2,24 +2,64 @@
 $default-font: 'Fira Sans', Helvetica, Arial, sans-serif;
 $header-font: 'Fira Sans', Helvetica, Arial, sans-serif;
 
+// default css variables
+:root {
+  --color-grey-lightest: #fbfbfb;
+  --color-grey-lighter: #ebebeb;
+  --color-grey-light: #d4d4d4;
+  --color-grey: #b1b1b1;
+  --color-grey-dark: #858585;
+  --color-grey-disabled: #999999;
+
+  --color-blue: #0996f8;
+  --color-blue-dark: #0670cc;
+  --color-blue-darker: #005bab;
+  --color-blue-darkest: #004480;
+
+  --color-blue-focusRing: hsla(208,100%,69%,.75);
+
+  --color-green-lightest: #E2FDDC;
+  --color-green: #57BD35;
+  --color-green-dark: #40A624;
+  --color-green-darker: #399320;
+  --color-green-darkest: #2E761A;
+
+  --border-radius-osx: 4px;
+  --border-radius-outer: 4px;
+  --border-radius-inner: 3px;
+}
+
 //Color Variables
-$button-background-color: #0095dd;
-$button-background-disabled-color: #8a9ba8;
-$button-background-hover-color: #08c;
+$button-background-color: var(--color-blue);
```

Comment:
@shane-tomlinson No it doesn't. So I was following the moz-style-guide by @shorlander here: http://people.mozilla.org/~shorlander/styleguide/ui-elements/buttons-all.html and it was using css variables. I could definitely just use sass variables instead. Should I do that?